### ISSUE TITLE
Clear Room History

### ISSUE THREAD

**Issue by griffins-planera** [liveblocks/liveblocks#1117]
**Is your feature request related to a problem? Please describe.**

Can we be allowed to clear room history (undo/redo stack)? Similarly to a computer's file directory, a user in our product can navigate to multiple UI levels of the same project. Currently the undo stack will build up multiple actions from one level, and once the user changes levels the undo stack will affect offscreen elements. The ability to reset history would allow us to ensure the undo stack is relevant only to the view the user is currently on. 

**Describe the solution you'd like**

Please expose a function to clear room history adjacent to `history.undo()` and `history.redo()`. Perhaps `history.clear()`

**Additional context**

We are using the REACT library.

**Comment by nvie** [comment]
Hi @griffins-planera – yes, we want to offer this API. It's on our roadmap, but we haven't started the work on this yet.

**Comment by griffins-planera** [comment]
@nvie Thanks, do you have a link to a product roadmap? Otherwise, where is the best place to track the status of the feature? Here for now?

**Comment by nvie** [comment]
Our roadmap is not public (yet). But since this was just a small feature and we wanted to add support for it anyway, I just [implemented it](https://github.com/liveblocks/liveblocks/pull/1125) just now.

**Comment by CTNicholas** [comment]
Hey @griffins-planera, we've just released Liveblocks 1.3, and this is now possible!

```ts
import { useHistory } from "../liveblocks.config";

function App() {
  const history = useHistory();

  return (
    <button onClick={history.clear}>
      Clear undo/redo history
    </button>
  );
}
```

You need to update all your Liveblocks packages to the same version, e.g.:
```bash
npm i @liveblocks/client@latest @liveblocks/react@latest [+ any other Liveblocks packages you're using]
```

API: https://liveblocks.io/docs/api-reference/liveblocks-client#Room.history.clear

**Comment by griffins-planera** [comment]
@nvie @CTNicholas Awesome! Much appreciated
